x/inter-tx/types/msgs.go

```diff
@@ -52,3 +53,58 @@ func (msg MsgRegisterAccount) GetSigners() []sdk.AccAddress {
 	}
 	return []sdk.AccAddress{accAddr}
 }
+
+var _ sdk.Msg = &MsgSend{}
+
+// NewMsgSend creates a new MsgSend instance
+func NewMsgSend(
+	chainType, port, channel string, sender, toAddress sdk.AccAddress, amount []sdk.Coin,
```

Comment:
I would advise using a `string` to represent the addresses. I don't think this [issue](https://github.com/cosmos/cosmos-sdk/issues/8332) has been fixed yet 

I'm not entirely sure what the ramifications are, but it is good to be aware of. Using a `string` to represent the address is a safe bet

Reply:
Would you still recommend this or can we resolve as discussed here -> https://github.com/cosmos/interchain-account/pull/27#discussion_r601528607

Reply:
Thanks for linking the issue re: String()